tests/test_marathon_serviceinit.py

```diff
@@ -429,26 +429,74 @@ def test_status_smartstack_backends_normal():
 def test_status_smartstack_backends_different_nerve_ns():
     service = 'my_service'
     instance = 'my_instance'
-    cluster = 'fake_cluster'
     different_ns = 'other_instance'
```

Comment:
Don't call it "other_instance" just call it "different_ns" to make the test read more obviously.

In this context it isn't really another instance, it is just another nerve namespace.